<launch>
  <!-- Parameters -->
  <rosparam file="$(find traversability_mapping)/config/params.yaml" command="load" subst_value="true" />

  <!-- Terrain Mapping -->
  <node pkg="traversability_mapping" type="terrain_mapping_node" name="terrain_mapping" output="screen" />

  <!-- Rviz -->
  <node pkg="rviz" type="rviz" name="rviz_terrain_mapping" args="-d $(find traversability_mapping)/rviz/terrain_mapping.rviz" />

  <!-- RQT -->
  <!-- <node pkg="rqt_gui" type="rqt_gui" name="rqt" args="perspective-file $(find wavefront_planner)/rviz/rviz.perspective" /> -->
</launch>